services/slam/builtin/cartographer_int_test.go

```diff
@@ -63,10 +67,14 @@ func testCartographerInternalState(t *testing.T, svc slam.Service, dataDir strin
 	internalState, err := svc.GetInternalState(context.Background(), "test")
 	test.That(t, err, test.ShouldBeNil)
 
-	// Save the data from the call to GetInternalState for use in next test.
+	internalStateStream, err := slam.GetInternalStateFull(context.Background(), svc, "test")
+	test.That(t, err, test.ShouldBeNil)
+	test.That(t, len(internalState), test.ShouldEqual, len(internalStateStream))
```

Comment:
I dont think this test is necessary as the `GetInternalState` function will be disappearing soon